dashi/poolqc/pooling_qc_sample.py

```diff
@@ -18,70 +20,150 @@
 bcl2fastq = gsiqcetl.bcl2fastq.parse.load_cache(
     gsiqcetl.bcl2fastq.parse.CACHENAME.SAMPLES,
     './data/bcl2fastq_cache.hd5')
+bcl2fastq.rename(columns={'SampleNumberReads': 'Clusters'}, inplace=True)
 bcl2fastq['library'] = bcl2fastq['SampleID'].str.extract('SWID_\d+_(\w+_\d+_.*_\d+_[A-Z]{2})_')
 
 df = bcl2fastq.merge(rnaseq, on='library', how='outer')
 df = df.merge(bamqc, on='library', how='outer')
 df = df.drop(columns=['Sequencer Run Name', 'Lane Number'])
 
-runs = df['Run'].dropna().sort_values(ascending=False).unique()
-
-# Currently using arbitrary values as threshold for visualisation
-INDEX_THRESHOLD = 500000
+runs = df['Run'].sort_values(ascending=False).unique()
+runs = [x for x in runs if str(x) != 'nan']
 
 layout = html.Div(children=[
-    dcc.ConfirmDialog(
-        id='warning',
-        message='The selected run does not return any data. Analysis may have not been completed yet.' '''
+
+    sd.Drawer(id='Filter_drawer',
+              width='40%',
+              docked=False,
+              children=[
+                  html.Div([html.P(children='Report Filters', style={'font-size': '30px', 'padding-top': '10px',
+                                                                     'text-align': 'center'}),
+                            sd.Divider(),
+                            html.P(children='Select a Run', style={'padding-left': '5px', 'font-weight': 'bold'}),
+                            dcc.Dropdown(
+                                id='select_a_run',
+                                options=[{'label': r, 'value': r} for r in runs],
+                                value=runs[0],
+                                clearable=False
+                            ),
+                            html.Br(),
+                            html.P(children='Select a Lane', style={'padding-left': '5px', 'font-weight': 'bold'}),
+                            html.Div(
+                                dcc.RadioItems(
+                                    id='lane_select',
+                                    labelStyle={'display': 'inline-block', 'padding-left': '30px', }
+                                )
+                            ),
+                            html.Br(),
+                            html.P(children='Enter a Threshold Value for the Index Clusters',
+                                   style={'padding-left': '5px', 'font-weight': 'bold'}),
+                            html.Div(
+                                dcc.Input(
+                                    id='index_threshold',
+                                    placeholder='Press "Enter" when complete',
+                                    debounce=True,
+                                    type='number',
+                                    value='0'
+                                )
+                            ),
+                            html.Br(),
+                            html.P(children='Select Which Type of Sample to View',
```

Comment:
This is hard to disambiguate with "Select a Sample Type" below. Can it be updated to "QC Status" or similar?